portfolio/src/main/webapp/script.js

```diff
@@ -18,15 +18,21 @@ function loadPage() {
   getComments();
 }
 
+//TO DO Change the += format to use Node.appendChild()
```

Comment:
nit: Please leave a blank space after the `//` and use a single word for TODO :)
It's also useful to assign the TODO to someone specially in a shared project so it'd be clear who should do it, e.g. `TODO(username): add more details`

Reply:
Will do, thanks!